x/qgb/orchestrator/orchestrator_client.go

```diff
@@ -244,63 +258,57 @@ func (oc *orchestratorClient) addOldDataCommitmentAttestations(
 ) {
 	oc.logger.Info("Started adding old Data Commitments attestation to queue")
 	defer oc.logger.Info("Finished adding old Data Commitments attestation to queue")
+	// FIXME I guess we can remove this until we have slashing. The reason being, without slashing
+	// it would be good to just sign everything. So that all attestations are signed.
```

Comment:
that could lead to a lot of waste, especially for new validators who don't need to sign. imo we should definitely keep this. Do we check if we've already signed old ones somewhere?

Reply:
yes, even if we want to sign again, it will raise an exception. What I mean is that since we don't have slashing, all orchestrators should sign the whole history, especially for testnets...  But this is to be discussed really, doesn't need to be included in this PR

Reply:
Done https://github.com/celestiaorg/celestia-app/issues/479